values/wire-server/prod-values.example.yaml

```diff
@@ -28,7 +28,7 @@ brig:
     elasticsearch:
       host: elasticsearch-external
     rabbitmq:
-      host: rabbitmq-external
+      host: rabbitmq
```

Comment:
Maybe, leave a comment here about the other possible value? (`rabbitmq-external`)

Reply:
fixed